src/devices/cpu/z80/z80.lst

```diff
@@ -142,16 +138,16 @@ macro   jr_cond
 
 macro   arg16_call
 	call arg16
-	m_ea = TDAT; TADR = PC-1;
-	call nomreq_addr
+	m_ea = TDAT;
+	call nomreq_addr PC-1
```

Comment:
I'm thinking of making macro call like `@nomreq_addr PC-1` (sometimes late)
Speak up if anybody against it.